persistence/genesis.go

```diff
@@ -321,3 +321,30 @@ func (p PostgresContext) GetAllFishermen(height int64) (f []modules.Actor, err e
 	}
 	return
 }
+
+// IMPROVE: This is a proof of concept. Ideally we should have a single query that returns all actors.
+func (p PostgresContext) GetAllStakedActors(height int64) (actors []modules.Actor, err error) {
+	actors = make([]modules.Actor, 0)
```

Comment:
if we don't initialize this, does it break?

Reply:
Oh, well, good to know! Thanks @Olshansk 🙏

https://go.dev/play/p/GjBg0QuZEbB

![image](https://user-images.githubusercontent.com/29378614/207729295-87d87521-4397-4487-8b3d-5967605423c7.png)
![image](https://user-images.githubusercontent.com/29378614/207729324-027286f4-46b5-4e27-a88b-d55568b1638d.png)

Nice

Reply:
![200w](https://user-images.githubusercontent.com/1892194/207743113-643b26f5-e7a1-4991-9699-35d0a6e4f81e.gif)